src/view/renderer.js

```diff
@@ -580,6 +582,17 @@ export default class Renderer {
 			return;
 		}
 
+		if ( oldViewSelection && this._areSimilarSelections( oldViewSelection, this.selection ) ) {
+			const data = {
+				oldSelection: oldViewSelection,
+				currentSelection: this.selection
+			};
+
+			log.warn( 'renderer-selection-similar: Rendering selection was skipped due to its similarity to the previous selection.', data );
```

Comment:
Yes, it is... maybe we should think about something like e.g. `log.info` or introduce some config which turns off this log. But if we plan to remove it in the future anyway, maybe there is no sense in spending time on it.